en/docs/learn/integration-tutorials/transforming-message-content.md

```diff
@@ -340,97 +326,32 @@ Let's configure the API resource with the data transformation logic.
-        </td>
-      </tr>
-      <tr>
-        <td>Output Type</td>
-        <td>
-          Select <b>JSON</b>.
-        </td>
-      </tr>
-    </table>
-
-    <img src="{{base_path}}/assets/img/integrate/tutorials/message-transformation/data-mapper-properties.png">
+11.  Open API resource. Add a **Call mediator** next to Data mapper from the **Mediators** palette and select the `HospitalServicesEP` endpoint from the dropdown list. Click **Submit**.
 
-16.  Add a Call mediator from the **Mediators** palette and add the HospitalServicesEP endpoint from the **Defined Endpoints** palette to the empty box adjoining the Call mediator.
+      <img src="{{base_path}}/assets/img/learn/tutorials/transforming-message-content/add-call-mediator-for-transformation.png">
 
-      <img src="{{base_path}}/assets/img/integrate/tutorials/message-transformation/add-call-mediator-for-transformation.png">
+12. Add a **Respond mediator** next to the **Call mediator** to return the response from the health care service back to the client. Click **Submit**.
 
-17. Add a **Respond mediator** next to the **Call** mediator to return the response from the health care service back to the client.
-
-      <img src="{{base_path}}/assets/img/integrate/tutorials/message-transformation/add-respond-mediator-for-transformation.png">
+      <img src="{{base_path}}/assets/img/learn/tutorials/transforming-message-content/add-respond-mediator-for-transformation.png">
     
 18. Save the REST API configuration.
 
-You have successfully created all the artifacts that are required for this use case. 
-
-### Step 3: Package the artifacts
-
-Package the artifacts in your composite exporter module (SampleServicesCompositeExporter) to be able to deploy the artifacts in the server.
-
-1.  Open the `          pom.xml         ` file in the composite exporter module.
-2.  Ensure that the following projects and artifacts are selected in the POM file.
-
-    -   SampleServicesCompositeExporter
-        -   `HealthcareAPI`
-        -   `HospitalServicesEP`
-    -   SampleServicesRegistryResources
-
-3.  Save the changes.
-
-### Step 4: Build and run the artifacts
-
-To test the artifacts, deploy the [packaged artifacts](#step-3-package-the-artifacts) in the embedded Micro Integrator:
+You have successfully created all the artifacts required for this use case.
 
-1.  Right-click the composite exporter module and click **Export Project Artifacts and Run**.
-2.  In the dialog box that opens, confirm that the required artifacts from the composite exporter module are selected.     
-4.  Click **Finish**. 
+### Step 3: Build and run the artifacts
 
-The artifacts will be deployed in the embedded Micro Integrator and the server will start.
+{!includes/build-and-run-artifacts.md!}
 
-- See the startup log in the **Console** tab.
-- See the URLs of the deployed services and APIs in the **Runtime Services** tab.
-
-### Step 5: Test the use case
+### Step 4: Test the use case
 
```

Comment:
Open a terminal, navigate to the location where your saved the back-end service. ->

Open a terminal, navigate to the location where you have saved the back-end service.

Reply:
Add the request information as given below and click the <b>Send</b> button.
->

 Add the request information as shown below and click the <b>Send</b> button.

Reply:
There is still content which has integration studio.


Let's send a request to the API resource to make a reservation. You can use the embedded <b>HTTP Client</b> of WSO2 Integration Studio as follows:

Reply:
Please check for grammar in the entire page

Reply:
This JSON payload contains details of the appointment reservation, which includes patient details, doctor, hospital, and `data of appointment`.

Please correct this too.

Reply:
> There is still content which has integration studio.
> 
> Let's send a request to the API resource to make a reservation. You can use the embedded HTTP Client of WSO2 Integration Studio as follows:

for this should change it to "Micro Integrator extension" or remove the entire part of the embedded HTTP client because the below explains how to send HTTP requests using Postman.